backend/migrations/20210130150415_matches.js

```diff
@@ -0,0 +1,22 @@
+const { tableMatches } = require('../src/models/match')
+
+exports.up = (knex) => {
+    return knex.schema.createTable(tableMatches, (table) => {
+      table.increments('MatchID').primary()
+      table.integer('StudentID').references('StudentID').inTable('students').onDelete("CASCADE")
+      table.integer('TeacherID').references('TeacherID').inTable('teachers').onDelete("CASCADE")
```

Comment:
Shall we make the `StudentID/TeacherID` combo unique? I think it will be illegal to have 2 match records between the same student and teacher?

Reply:
Done.